pkgs/top-level/all-packages.nix

```diff
@@ -6282,7 +6282,6 @@ in
 
   volume_key = callPackage ../development/libraries/volume-key { };
 
-  vorbisgain = callPackage ../tools/misc/vorbisgain { };
```

Comment:
I don't think that this line is a good idea to remove ;)

Reply:
sorry, not sure how this happened :)

Reply:
fixed.